README.md

```diff
@@ -89,13 +89,36 @@ Assuming that we have `apps/foo/bar.py` file with the given codes. The commented
         # apps/foo/bar.py::Baz::method_1
         def method_1(self):
             pass
+
+- Pytest Format
```

Comment:
I think this should be `Import` Format since Pytest Format is already included above?
Also, please add the enclosing ``` markdown chars above since the previous section above was not closed properly due to copy-paste.

Reply:
Oops, thanks for spotting. Amended both issues in f56ccf3d76a00fe79f3a85363aeb24345be909ef:

https://github.com/ranelpadon/python-copy-reference.vim/blob/f56ccf3d76a00fe79f3a85363aeb24345be909ef/README.md?plain=1#L92

https://github.com/ranelpadon/python-copy-reference.vim/blob/f56ccf3d76a00fe79f3a85363aeb24345be909ef/README.md?plain=1#L94